### PR TITLE
Added FAQ information

### PR DESCRIPTION
**Pull request by ralfjansson**
Copied the FAQ information from the wiki and added it to the new docs.

### CODE REVIEW

File: Documentation/monogame_faq.md [hunk 1]
```diff
@@ -3,5 +3,135 @@ This page contains a list of frequently asked questions.
...
+ - iOS - You need Xamarin.iOS: [http://ios.xamarin.com/], it works on Windows and Mac and it can be used in combination with either Visual Studio or Xamarin Studio.
+
+
+### Where do I start? 
+
+The quickest way to start is to download CartBlanche's samples: [https://github.com/CartBlanche/MonoGame-Samples https://github.com/CartBlanche/MonoGame-Samples]
+
+### Where is the IRC channel? 
+
+The official MonoGame IRC channel is #MonoGame on irc.gnome.org.
```
Comment (cra0zy) [inline Documentation/monogame_faq.md]: I don't think we are using the IRC channel with Gitter around.

**Comment by tomspilman** [comment]
@ralfjansson - The reason this failed is we changed our build process and that required a couple of changes in the repo.  Your branch is over 200 commits behind the develop branch... so you don't have those changes.

Just merge the latest develop into your fork or rebase and this will build correctly.

File: Documentation/monogame_faq.md (hunk 1 above)
Comment (tomspilman) [inline Documentation/monogame_faq.md]: Yeah...  I don't know if anyone is on there anymore.  If there are still people on there then I see no problem referencing it, but we should first suggest the Gitter channel.  https://gitter.im/mono/MonoGame

**Comment by tomspilman** [comment]
@dellis1972 - Can you review the iOS questions in the FAQ.

**Comment by cra0zy** [comment]
@ralfjansson Do the following:
```
git remote add upstream https://github.com/mono/MonoGame.git
git fetch upstream
git rebase upstream/develop
git push --force
```

**Comment by dellis1972** [comment]
@tomspilman I'll take a look. but @CartBlanche might be better suited. He's done more iOS stuff that me... I think

File: Documentation/monogame_faq.md [hunk 2]
```diff
@@ -3,5 +3,135 @@ This page contains a list of frequently asked questions.
...
 Depending on the platform you wish to develop for the following thing are needed:
- - Android - You need Xamarin.Android: http://android.xamarin.com/, it works on Windows and Mac and it can be used in combination with either Visual Studio or Xamarin Studio.
- - iOS - You need Xamarin.iOS: http://ios.xamarin.com/, it works on Windows and Mac and it can be used in combination with either Visual Studio or Xamarin Studio.
+ - Android - You need Xamarin.Android: [http://android.xamarin.com/], it works on Windows and Mac and it can be used in combination with either Visual Studio or Xamarin Studio.
+ - iOS - You need Xamarin.iOS: [http://ios.xamarin.com/], it works on Windows and Mac and it can be used in combination with either Visual Studio or Xamarin Studio.
+
+
+### Where do I start? 
+
+The quickest way to start is to download CartBlanche's samples: [https://github.com/CartBlanche/MonoGame-Samples https://github.com/CartBlanche/MonoGame-Samples]
```
Comment (dellis1972) [inline Documentation/monogame_faq.md]: I think we are moving to https://github.com/MonoGame/MonoGame.Samples
The CartBlanche ones are very old and not up to date.

File: Documentation/monogame_faq.md [hunk 3]
```diff
@@ -3,5 +3,135 @@ This page contains a list of frequently asked questions.
...
+The official MonoGame IRC channel is #MonoGame on irc.gnome.org.
+
+### How can I take advantage of iOS 4 Multitasking ? 
+
+Your Program class has to extend MonoGameProgram instead of UIApplicationDelegate. Assign MonoGameGame variable to your Game class.
+
+```C#
+[Register ("AppDelegate")]
+	class  Program : MonoGameProgram
+	{
```
Comment (dellis1972) [inline Documentation/monogame_faq.md]: MonoGameProgram doesn't appear to exist any more....

**Comment by Jjagg** [comment]
This is very outdated stuff, closing.

**Pull request closed without merging**